sympy/tensor/array/arrayop.py

```diff
@@ -208,3 +208,38 @@ def derive_by_array(expr, dx):
             return ImmutableDenseNDimArray([expr.diff(i) for i in dx], dx.shape)
         else:
             return diff(expr, dx)
+
+
+def permutedims(expr, perm):
+    """
+    Permutes the indices of an array.
+
+    Parameter specifies the permutation of the indices.
+    """
+    if not isinstance(expr, NDimArray):
+        raise TypeError("expression has to be an N-dim array")
+
+    perm = tuple(perm)
```

Comment:
@Upabjojr are you sure?

I am not familiar with `Permutation` class, so I tried to compare its behaviour with `numpy.transpose`, and got the same results for both.
Example:

```python
In [1]: import numpy as np

In [2]: from sympy.combinatorics import Permutation

In [3]: shape = (5,6,7,8)

In [4]: perm = (1,3,0,2)

In [5]: x = np.zeros(shape)

In [6]: np.transpose(x, axes = perm).shape
Out[6]: (6, 8, 5, 7)

In [7]: Permutation(perm)(shape)
Out[7]: [6, 8, 5, 7]
```

Could you provide any counter-example for it?